src/parameter_entry_class.F90

```diff
@@ -488,7 +493,7 @@ subroutine get_vector_character (this, value, errc)
     type is (character(*))
 #ifdef INTEL_DPD200255963
       if (allocated(value)) deallocate(value)
-      allocate(value(size(v)), source=v)
+      allocate(character(len(v(1))) :: value(size(v)))
```

Comment:
Ah, it looks like it might be fixed in gfortran 5.2, I haven't tried that one yet.